src/test/java/org/apache/commons/collections4/iterators/LoopingListIteratorTest.java

```diff
@@ -21,8 +21,8 @@
 import java.util.List;
 import java.util.NoSuchElementException;
 
-import org.junit.Test;
-import static org.junit.Assert.*;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
@arturobernalg 
Did you forget to flip the arguments on the `assert*` API calls in this file and subsequent files?

Reply:
HI @garydgregory 

Nop. I don't.  I working on change/migrate the rest of the class. Trying to find out how migrate the BulkTest that extends TestCase hehehe. 

Any advice is more than welcome

Reply:
I don't have specific advice, that is some older code initially based on JUnit 3.

Reply:
I can imagine. My idea it's split the migration in several PR.  In that way i can reduce the number of file to review and reduce the errors